xilem_core/src/views/memoize.rs

```diff
@@ -116,15 +119,123 @@ where
 }
 
 /// Memoize the view, until the `data` changes (in which case `view` is called again)
-pub fn memoize<State, Action, Context, Message, Data, V, ViewFn>(
+pub fn memoize<State, Action, Context, Message, Data, V, InitView>(
     data: Data,
-    view: ViewFn,
-) -> Memoize<Data, ViewFn>
+    init_view: InitView,
+) -> Memoize<Data, InitView, State, Action>
 where
     Data: PartialEq + 'static,
-    ViewFn: Fn(&Data) -> V + 'static,
+    InitView: Fn(&Data) -> V + 'static,
     V: View<State, Action, Context, Message>,
     Context: ViewPathTracker,
 {
-    Memoize::new(data, view)
+    Memoize::new(data, init_view)
+}
+
+/// Specialized version of [`Memoize`], which doesn't take any data at all, the closure is evaluated only once and when a child view forces a rebuild
+pub struct Static<InitView, State, Action> {
+    init_view: InitView,
+    phantom: PhantomData<fn() -> (State, Action)>,
+}
+
+/// Specialized version of [`memoize`], which doesn't take any data at all, the closure is evaluated only once and when a child view forces a rebuild
+pub fn static_view<State, Action, Context, Message, V, InitView>(
```

Comment:
I thought against this in Bevy, and I'm going to fight against this here; we do not want components to have names ending in `_view` .

I know that we can't use `static` because that's a keyword. Maybe `eternal` works?

Reply:
`eternal` sounds epic at least^^ Yeah I'm also not happy with `static_view` ideal would be `static` or something. Though `eternal` probably suggests that this will live forever...

Reply:
`static` also suggests it will live forever. We definitely can't end up in a state where this is `_view` :sweat_smile:

Reply:
Also true... Yeah finding good names is hard, probably the hardest thing in programming^^ I'll try to think about it, but I agree, we should remove `_view`.

Reply:
I like `frozen` and `immutable`.
`fixed` reminds me more of positioning like in CSS `position: fixed`